app/(gcforms)/[locale]/(form filler)/id/[...props]/lib/notifications.ts

```diff
@@ -0,0 +1,246 @@
+import { sendEmail } from "@lib/integration/notifyConnector";
```

Comment:
If this file will live in the `app` directory please make sure it has the `"use server"` directive at the top.